fix: Improve AzureMonitorScraper Error Handling for Time Series Missing Requested Dimension Value (#2345)
tomkerkhove/promitor: changelog/content/experimental/unreleased.md

- {{% tag changed %}} Switch to Mariner distroless base images
- {{% tag security %}} Patch for [CVE-2023-29331](https://github.com/advisories/GHSA-555c-2p6r-68mm) (High)

> - {{% tag fixed %}} Better handle time series missing requested dimension in AzureMonitorClient [#2331](https://github.com/tomkerkhove/promitor/issues/2331))

Reviewer: Let's move this to the top of the list + improve wording:
```suggestion
- {{% tag fixed %}} Improve handling of time series with missing dimensions that are requested ([#2331](https://github.com/tomkerkhove/promitor/issues/2331))
```
